kaldb/src/test/java/com/slack/kaldb/chunk/ReadOnlyChunkImplTest.java

```diff
@@ -153,7 +153,7 @@ public void shouldHandleChunkLivecycle() throws Exception {
     assertThat(searchMetadataStore.getCached().get(0).snapshotName).isEqualTo(snapshotId);
     assertThat(searchMetadataStore.getCached().get(0).url)
         .isEqualTo("gproto+http://localhost:8080");
-    assertThat(searchMetadataStore.getCached().get(0).name).isEqualTo("localhost:8080");
+    assertThat(searchMetadataStore.getCached().get(0).name).isEqualTo(snapshotId + "_localhost");
```

Comment:
I think it's best to call the toName method here to check the name.

Reply:
Done.